src/wrap-store/wrapStore.js

```diff
@@ -71,25 +74,33 @@ export default (store, {
       return;
     }
 
-    /**
-     * Send store's current state through port
-     * @return undefined
-     */
-    const sendState = () => {
-      port.postMessage({
-        type: STATE_TYPE,
-        payload: store.getState()
-      });
+    let prevState = store.getState();
+
+    const patchState = () => {
+      const state = store.getState();
+      const diff = shallowDiff(prevState, state);
+
+      if (Object.keys(diff.updated).length || diff.removed.length) {
```

Comment:
This could be simplified to just `if (diff.length) {...}` if we tweak how the diff is implemented as mentioned above.